README.md

```diff
@@ -937,12 +937,15 @@ constexpr void swap(static_vector<T, N>& x,
   noexcept(noexcept(x.swap(y)));
 ```
 
-> - _Constraints_: This function shall not participate in overload resolution
->   unless `is_swappable_v<T>` is `true`.
+> - _Mandates_: `is_swappable_v<T>` is `true` and `is_move_constructible_v<T>` is `true`.
```

Comment:
I'd say "yes" to `is_move_constructible_v<T>`, and "no" to "Mandates". "Mandates" here will break `is_swappable_v<static_vector<T, N>>` and `Swappable<static_vector<T, N>>`.

Reply:
I can't comment on it because it's too far from the proposed changes, but the "*Remarks:* Shall not participate in overload resolution..." on 927 (and all similar *Remarks:*) should be a *Constraints:* element.

Reply:
I find not breaking `is_swapable` convincing.  Stay with *Constraints* then.